app/views/projects/tasks/_summary.html.erb

```diff
@@ -0,0 +1,38 @@
+<section class="summary-panel">
+  <h4 class="header-underline">My Tasks
+    <span class="actions pt-1">
+      <div class="action">
+        <%= link_to project_tasks_path(current_project) do %>
+          <i class="fa-solid fa-tasks sidebar-link-icon"></i> All Tasks
+        <% end %>
+      </div>
+    </span>
+  </h4>
+  <% if @tasks.any? %>
+    <ul class="list-group tasks">
+      <% @tasks.first(@tasks_limit).each do |task| %>
+        <li class="d-flex flex-column list-group-item">
+          <%= link_to project_board_list_card_path(current_project, task.board, task.list, task), class: "list-group-item-action text-truncate" do %>
+            <%= task.name %>
+          <% end %>
+          <span class="small text-muted"><%= task.board.name %></span>
+          <%= due_date_badge(task) %>
+        </li>
+      <% end %>
+    </ul>
+
+    <p class="small mt-2 mb-0 text-center">&nbsp;
+      <% if @tasks.size > @tasks_limit %>
+        + <%= @tasks.size - @tasks_limit %> more
```

Comment:
This could also be done in the controller or the view helper using something like `overflow_tasks`

`overflow_tasks.any?` /  `overflow_tasks.count`